docs/src/routes/theme/index.js

```diff
@@ -0,0 +1,201 @@
+// Dependencies
+import { h, Component } from "preact";
+
+// Material Components
+import Button from "preact-material-components/Button";
+
+// Components
+import ComponentTable from "../../components/component-table";
+import CodeBlock from "../../components/code-block";
+
+// Styles
+import style from "./style";
+
+// Class
+export default class ThemePage extends Component {
+  render() {
+    return (
+      <div className={style["page-theme"]}>
+        <div className="mdc-typography--display1">Original documentation</div>
+        <div className="mdc-typography--body">
+          preact-material-components brings{" "}
+          <span className="strong">mdc-theme</span>as is, you can refer to its
```

Comment:
I'll admit I haven't tested the PR, but isn't there a space missing after ```<span className="strong">mdc-theme</span>```? Maybe the style has a margin though, so not sure.

Reply:
fixed